src/component/race/css/Race.css

```diff
@@ -0,0 +1,63 @@
+.position0 {
+  padding-left: 0em;
+}
+
+.position1 {
+  padding-left: 2em;
+}
+
+.position2 {
+  padding-left: 4em;
+}
+
+.position3 {
+  padding-left: 6em;
+}
+
+.position4 {
+  padding-left: 8em;
+}
+
+.position5 {
+  padding-left: 10em;
+}
+
+.position6 {
+  padding-left: 12em;
+}
+
+.position7 {
+  padding-left: 14em;
+}
+
+.position8 {
+  padding-left: 16em;
+}
+
+.position9 {
+  padding-left: 18em;
+}
+
+.position10 {
+  padding-left: 20em;
+}
+
+.position11 {
+  padding-left: 22em;
+}
+
+.position12 {
+  padding-left: 24em;
+}
+
+.position13 {
+  padding-left: 26em;
+}
+
+.position14 {
+  padding-left: 28em;
+}
+
+.position15 {
+  padding-left: 30em;
+}
```

Comment:
As you pointed out, this isn't great.

Instead we could use in line styles to make things dynamic.

```
<div style={{ padding-left: `{<padding value>}` }} />
```